hsl_builder/Elements/CarouselItem.py

```diff
@@ -0,0 +1,48 @@
+from .Actionable import Actionable
+from typing import List
+
+class CarouselItem(object):
+    """
+    Create Carousel Items for Carousel
+
+    Attributes
+    ----------
+    title : str
+        Title of specific carousel item.
+        
+    subtitle : str
+        Subtitle of specific carousel item.
+        
+    description : str
+        More verbose description of item that is shown below the subtitle
+
+    actionables : List[Actionable]
+        Actionables to be performed when the user taps on the item
+
+    meta : str
+        meta value for the corresponding carousal item
+    """
+
+
+    def __init__(self,title: str, subtitle: str):
+        self.title: str = title
+        self.subtitle: str = subtitle
+        self.description: str = ''
+        self.thumbnail: str = ''
+        self.actionables: List[Actionable] = []
+        self.meta: str = ''
```

Comment:
Where are the setters and getters for this variables? Or they will be accessed directly?